anautilities.py

```diff
@@ -66,7 +66,7 @@ def getDirByAnaType(anaType, cName, ztrim=4):
         pass
 
     # Check Paths
-    from gempython.utils.wrappers import envCheck
+    from ..utils.wrappers import envCheck
```

Comment:
Probably my ignorance here but I don't really follow what this (change to line 69) does; or the motivation for it?

I guess it's related to the discussion:

>Update imports syntax in anautilities

> Update to anautilities has not been, but is a semantic change only, a proof-of-principle of proper relative imports that will be extended to other package specific modules (where appropriate, e.g., not in tools that utilize a __main__).

Reply:
`anautilities.py` lives in `gempython/gemplotting/`
It wants to `import` from `wrappers.py` which lives in `gempython/utils`
Because `anautilities.py` will **never** be called standalone (no `__main__`), it is a "part of the package", so it should reference all `import`s from the `gempython` namespace relatively (wrt where it lives) so it imports from `..utils.wrappers` (i.e., look in the package structure in the directory above where I am for `utils/wrappers.py`)
This type of `import` *should* (and **can**) only be done as deep as the package itself is (don't `import .....somewhere.outside.of.gempython`), and **must** not be done for anything that will be called either as a standalone, lives-on-the-`PATH` executable, nor for a module with a `__main__` (executable via `python -m <modulename>`, as those pieces are not a "part" of the package in the same sense (and can be located outside of the package itself)

Reply:
Ah so the `import ..some.package`

Is like calling `cd ..`.  I understand.